modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2564,12 +2672,6 @@ private void _testUpdateObjectEntryExternalReferenceCode()
 				"listTypeEntryKeyRequired", "listTypeEntryKey1"
 			).build());
 
-		Assert.assertNotNull(
-			ReflectionTestUtil.getFieldValue(objectEntry1, "_values"));
-		Assert.assertEquals(
-			String.valueOf(objectEntry1.getUuid()),
-			objectEntry1.getExternalReferenceCode());
-
```

Comment:
@p-albuquerque why did you remove this code?

Reply:
I'm doing this same asserts in another place in `testAddObjectEntry`